dependencies_with_url.csv

```diff
@@ -52,6 +52,7 @@ javax.xml.stream:stax-api:jar:1.0-2:compile,COMMON DEVELOPMENT AND DISTRIBUTION
 jline:jline:jar:0.9.94:compile,BSD,http://jline.sourceforge.net
 junit:junit:jar:4.12:compile,Eclipse Public License 1.0,http://junit.org
 junit:junit:jar:4.4:compile,Common Public License Version 1.0,http://junit.org
+junit:junit:jar:4.10:compile,Common Public License Version 1.0,http://junit.org
```

Comment:
I'll double check these. This might be able to be cleaned up, but I know in at least one case (json-simple), we're pulling tests in because they're compile in the transitive dependencies. https://jira.apache.org/jira/browse/METRON-2311

Reply:
`json-simple` does pull in 4.10. I assume that was just getting hidden by a different version getting pulled in from who knows where. 4.12 is pulled in by the migration support for `@Rule`, so it's junit5 itself pulling it in (for now).

I did dump 4.4 though